api/valor_api/backend/core/llm_clients.py

```diff
@@ -26,6 +26,87 @@ class Messages(BaseModel):
     messages: list[Message]
 
 
+def _generate_claims_instruction(text: str) -> str:
+    """
+    Instruction template was copied from DeepEval's codebase https://github.com/confident-ai/deepeval/blob/main/deepeval/metrics/faithfulness/template.py.
+
+    Parameters
+    ----------
+    text: str
+        The text to extract claims from.
+
+    Returns
+    -------
+    str
+        The instruction for the llm.
+    """
+    return f"""Based on the given text, please generate a comprehensive list of FACTUAL claims that can inferred from the provided text.
+
+**
```

Comment:
can all of this text be indented similarly?

Reply:
Indenting would make the actual instruction indented. In DeepEval, they don't indent the instruction (for example https://github.com/confident-ai/deepeval/blob/main/deepeval/metrics/faithfulness/template.py).